generate_token.py

```
@@ -0,0 +1,51 @@
+                raise ValueError("Token generation failed: No JWT returned.")
+
+            return token.jwt
+
+
+    except ValueError:
+
+       logging.exception("Token generation failed: no JWT returned.")
+
+       raise
+
+    except (ConnectionError, TimeoutError) as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)